wait_for_it/test_wait_for_it.py

```diff
@@ -96,9 +96,18 @@ def test_service_available(self, _label, extra_argv):
         finally:
             server.stop()
 
-    @parameterized.expand([("parallel", ["-p"], 2), ("serial", [], 1)])
-    def test_service_unavailable(self, _label, extra_argv, expected_report_count):
-        host, port, sock = _occupy_free_tcp_port()
+    @parameterized.expand(
+        [
+            ("parallel", ["-p"], 2, "127.0.0.1"),
+            ("parallel", ["-p"], 2, "0.0.0.0"),
+            ("parallel", ["-p"], 2, "localhost"),
+            ("serial", [], 1, "127.0.0.1"),
+            ("serial", [], 1, "0.0.0.0"),
+            ("serial", [], 1, "localhost"),
```

Comment:
PS: This could use `functools.product` if you like. Just an idea.

Reply:
Thanks for the suggestion but I prefer to be explicit.